doc/release/RELEASE-NOTES.md

```diff
@@ -4,6 +4,36 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). The format is based on the `RELEASE-NOTES-TEMPLATE.md` file.
 
 
+## Release 1.17.0
+
+## Introduction
+* Product name: Open Supply Hub
+* Release date: *Provide release date*
```

Comment:
Please provide the release date. It is July 27, 2024.
You can find it in the `RELEASE-PROTOCOL.md`.

Reply:
I didn't know about it. Thank you for showing it to me.